docs/reference/setup/sysconfig/virtual-memory.asciidoc

```diff
@@ -1,8 +1,8 @@
 [[vm-max-map-count]]
 === Virtual memory
 
-Elasticsearch uses a <<default_fs,`hybrid mmapfs / niofs`>> directory by
-default to store its indices.  The default operating system limits on mmap
+Elasticsearch uses a <<default_fs,`mmapfs`>> directory by
+default for 64bit systems to store its indices.  The default operating system limits on mmap
```

Comment:
actually we do not support 32-bit systems so I think we can just say we use `mmapfs` all the time